cmd/kaf/group.go

```diff
@@ -256,13 +259,7 @@ var groupLsCmd = &cobra.Command{
 			fmt.Fprintf(w, "%v\t%v\t%v\t\n", detail.GroupId, state, consumers)
 		}
 
-		if len(groupDescs) != 0 {
-			w.Flush()
-		} else {
-			fmt.Printf("No Groups found\n")
-		}
```

Comment:
This was the only command, that returns message string if nothing is found. It was a problem for parsing output, so I just deleted it.